react/components/AccordionFilterContainer.js

```diff
@@ -48,40 +49,48 @@ class AccordionFilterContainer extends Component {
   }
 
   render() {
-    const { filters, intl, getLinkProps, map, rest } = this.props
+    const { filters, intl, filtersChecks, handleFilterCheck, selectedFilters } = this.props
     const { openedItem } = this.state
 
-    const nonEmptyFilters = filters.filter(spec => spec.options.length > 0)
-
+    let nonEmptyFilters = filters.filter(spec => spec.options.length > 0)
     return (
-      <Popup
-        title={intl.formatMessage({ id: 'search-result.filter-action.title' })}
-        id="filters"
-        icon={<FilterIcon size={16} active/>}
-      >
-        {({ onClose }) => (
-          <div className="vtex-accordion-filter">
-            {nonEmptyFilters.map(filter => {
-              const { type, title, options, oneSelectedCollapse } = filter
-              const isOpen = openedItem === filter.title
-              return (
-                <AccordionFilterItem
-                  key={filter.title}
-                  type={type}
-                  title={title}
-                  options={mountOptions(options, type, map, rest)}
-                  open={isOpen}
-                  show={openedItem === null ? true : isOpen}
-                  onOpen={this.handleOpen(filter.title)}
-                  onItemSelected={this.handleItemSelected(onClose)}
-                  oneSelectedCollapse={oneSelectedCollapse}
-                  getLinkProps={getLinkProps}
-                />
-              )
-            })}
+      <div className="vtex-accordion-filter">
+        <div className="pointer flex flex-row items-center pa5 h3 bg-base w-100 z-max bb b--muted-3 bw1">
+          <div className="c-muted-1 pv4 flex items-center" onClick={e => this.setState({ openedItem: null, })}>
```

Comment:
Why receiving the `event` on the onclick?

Reply:
removed